common/components/controllers/AboutProjectController.jsx

```diff
@@ -35,6 +35,11 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
       this.loadProjectDetails.bind(this),
       this.handleLoadProjectFailure.bind(this)
     );
+    ProjectAPIUtils.fetchProjectVolunteerList(
```

Comment:
I'm thinking we should initiate this call once the fetchProjectDetails call returns, to avoid flooding the server with too many simultaneous requests.

Reply:
Taking your opinion into account, I have modified and committed the code to implement volunteer list api call after the project call takes place and page render completes. 
Also taking the SEO factor into account, I ensured that prerender function executes after the volunteers list loads up so that the whole page content can be grabbed and returned to web crawlers. Please check this [webpage](https://docs.prerender.io/article/25-empty-or-partially-rendered-pages) for details.